src/utils/pgwire/src/pg_extended.rs

```diff
@@ -496,6 +506,12 @@ impl PreparedStatement {
                     };
                     format!("'{}'::INTERVAL", tmp)
                 }
+                _ => {
```

Comment:
Let's avoid using `_` here so we'll be noticed to update the logic here if new types are added in the future.

```suggestion
                ??, ?? => {
```

Reply:
Same for other occurrences.